blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DRawIO.java

```diff
@@ -0,0 +1,145 @@
+package com.rackspacecloud.blueflood.io.datastax;
+
+import com.codahale.metrics.Timer;
+import com.datastax.driver.core.*;
+import com.datastax.driver.core.querybuilder.Insert;
+import com.datastax.driver.core.querybuilder.Select;
+import com.rackspacecloud.blueflood.io.CassandraModel;
+import com.rackspacecloud.blueflood.io.Instrumentation;
+import com.rackspacecloud.blueflood.io.serializers.metrics.RawSerDes;
+import com.rackspacecloud.blueflood.types.DataType;
+import com.rackspacecloud.blueflood.types.IMetric;
+import com.rackspacecloud.blueflood.types.Locator;
+import com.rackspacecloud.blueflood.types.Range;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+import static com.datastax.driver.core.querybuilder.QueryBuilder.*;
+
+
+/**
+ * This class deals with writing boolean, numeric and string values to the metrics_full & metrics_string column
+ * families using the Datastax driver.
+ */
+public class DRawIO {
```

Comment:
My vote is to keep them the same as it encapsulates the Numeric vs String check in ```insertAsync```.  Otherwise I think we'd have to pull that check up, which doesn't really help with readability.